pxl.py

```diff
@@ -0,0 +1,99 @@
+class Pixel:
+
+    # The constructor that takes three arguments: red, green, and blue
+    def __init__(self, red, green, blue):
+        if not 0 <= red <= 255 or not 0 <= green <= 255 or not 0 <= blue <= 255:
+            raise ValueError("ValueError: pixel components must be in range [0, 255]!")
+        self._red = red
+        self._green = green
+        self._blue = blue
+
+    # Implemented property that allows read-only access to the private component field
+    @property
+    def red(self):
+        return self._red
+
+    # Implemented property that allows read-only access to the private component field
+    @property
+    def green(self):
+        return self._green
+
+    # Implemented property that allows read-only access to the private component field
+    @property
+    def blue(self):
+        return self._blue
+
+    # Adding two pixels results in a new Pixel object with components equal to the sum of the corresponding components
+    def __add__(self, other):
+        new_red = min(255, max(0, self.red + other.red))
+        new_green = min(255, max(0, self.green + other.green))
+        new_blue = min(255, max(0, self.blue + other.blue))
+        return Pixel(new_red, new_green, new_blue)
+
+    # Subtracting two pixels results in a new Pixel object with components equal to the difference of the
+    # corresponding components
+    def __sub__(self, other):
+        new_red = min(255, max(0, self.red - other.red))
+        new_green = min(255, max(0, self.green - other.green))
+        new_blue = min(255, max(0, self.blue - other.blue))
+        return Pixel(new_red, new_green, new_blue)
+
+    # Multiplying two pixels results in a new Pixel object with components equal to the multiplied of the
+    # corresponding components. Errors handled
```

Comment:
"Operator *(/) should allow multiply/divide Pixel object by integer or float number greater than zero."
There's no mentioning about multiplying/dividing pixel by other pixel, only by int/float